src/partials/contacts.html

```diff
@@ -0,0 +1,60 @@
+<meta name="robots" content="all" />
+
+<div class="contacts">
+  <div class="contacts__category">cafe</div>
+  <div class="contacts__localization">
+    <div class="contacts__place">Chicago</div>
+    <div class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</div>
+  </div>
+  <div class="contacts__availability">
+    <div class="contacts__days">Monday - Friday</div>
+    <div class="contacts__hours">06:00 AM - 10:00 PM</div>
+    <div class="contacts__days">Saturday - Sunday</div>
+    <div class="contacts__hours">08:00 AM - 16:00 PM</div>
+  </div>
+  <div class="contacts__communication">
+    <a class="contacts__phone" href="+61(0) 383 766 284">+61(0) 383 766 284</a>
+    <a class="contacts__email" href="noreply@envato.com">noreply@envato.com</a>
+  </div>
+</div>
+
+<div class="contacts">
+  <div class="contacts__category">foodtruck</div>
+  <div class="contacts__localization">
+    <div class="contacts__place">Los Angeles</div>
+    <div class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</div>
+  </div>
+  <div class="contacts__availability">
+    <div class="contacts__days">Monday - Friday</div>
+    <div class="contacts__hours">06:00 AM - 10:00 PM</div>
+    <div class="contacts__days">Saturday - Sunday</div>
+    <div class="contacts__hours">08:00 AM - 16:00 PM</div>
+  </div>
+  <div class="contacts__communication">
+    <a class="contacts__phone" href="+61(0) 383 766 284">+61(0) 383 766 284</a>
+    <a class="contacts__email" href="noreply@envato.com">noreply@envato.com</a>
+  </div>
+</div>
+
+<div class="contacts">
+  <div class="contacts__category">cafe</div>
+  <div class="contacts__localization">
+    <div class="contacts__place">New York</div>
+    <div class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</div>
+  </div>
+  <div class="contacts__availability">
+    <div class="contacts__days">Monday - Friday</div>
+    <div class="contacts__hours">06:00 AM - 10:00 PM</div>
+    <div class="contacts__days">Saturday - Sunday</div>
+    <div class="contacts__hours">08:00 AM - 16:00 PM</div>
+  </div>
+  <div class="contacts__communication">
+    <a class="contacts__phone" href="+61(0) 383 766 284">+61(0) 383 766 284</a>
+    <a class="contacts__email" href="noreply@envato.com">noreply@envato.com</a>
+  </div>
+</div>
+
+<div class="contacts__store">
+  <button class="contacts__store-button" type="button">Our Locations</button>
```

Comment:
w button jest jeszcze strzałka w prawo to jest jako svg, w button powinien się jeszcze znajdować tag svg